templates/appliance/hardware.html

```diff
@@ -0,0 +1,603 @@
+          </div>
+        </div>
+        <div class="col-4 p-card--layout">
+          <div class="p-card__header">
+            <h4>RPi Compute Module 3+</h4>
+            <div class="p-label--new">CERTIFIED</div>
+          </div>
+          <div class="p-card__main">
+            <div class="u-align--center">
+              {{
+                image(
+                  url="https://assets.ubuntu.com/v1/a7fd386a-Module-3+.jpg?w=225",
+                  alt="",
+                  height="149",
+                  width="225",
+                  hi_def=True,
+                  loading="auto"
+                ) | safe
+              }}
+            </div>
+            <ul class="p-list">
+              <li class="p-list__item">
+                1GB RAM
+              </li>
+              <li class="p-list__item">
+                1.2GHz BCM2837processor
+              </li>
+              <li class="p-list__item">
+                Fits a DDR2 SODIMM connector
+              </li>
+            </ul>
+          </div>
+          <div class="p-card__footer">
+            <button class="p-button u-no-margin--bottom" disabled="disabled">Discontinued</a>
+          </div>
+        </div>
+        <div class="col-4 p-card--layout">
+          <div class="p-card__header">
+            <h4>Raspberry Pi 2</h4>
+            <div class="p-label--new">CERTIFIED</div>
+          </div>
+          <div class="p-card__main">
+            <div class="u-align--center">
+              {{
+                image(
+                  url="https://assets.ubuntu.com/v1/fb8a736f-Pi2.jpg?w=225",
+                  alt="",
+                  height="143",
+                  width="225",
+                  hi_def=True,
+                  loading="auto"
+                ) | safe
+              }}
+            </div>
+            <ul class="p-list">
+              <li class="p-list__item">
+                1GB RAM
+              </li>
+              <li class="p-list__item">
+                Full-size HDMI
+              </li>
+              <li class="p-list__item">
+                USB 2.0 x4
+              </li>
+              <li class="p-list__item">
+                Micro-USB power supply
+              </li>
+              <li class="p-list__item">
+                40 pin GPIO
+              </li>
+            </ul>
+          </div>
+          <div class="p-card__footer">
+            <button class="p-button u-no-margin--bottom" disabled="disabled">Discontinued</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="u-fixed-width">
+        <hr class="p-separator" style="margin: 2rem 0;" />
+      </div>
+
+      <div class="row">
+        <h3 class="u-sv3">Certified appliances for Raspberry Pi</h3>
+      </div>
+
+      {% include "/appliance/shared/_featured_appliances.html" %}
```

Comment:
```suggestion
      <div class="u-hide--small">{% include "/appliance/shared/_featured_appliances.html" %}</div>
```

perhaps don't show this twice on mobile, just the bottom one?

Reply:
Why would we hide core information based on screen size?

Reply:
Oh its not in there twice but included in section so only one shown at a time